receiver/azuremonitorreceiver/config.go

```diff
@@ -242,11 +243,6 @@ type Config struct {
 	Cloud                             string                        `mapstructure:"cloud"`
 	SubscriptionIDs                   []string                      `mapstructure:"subscription_ids"`
 	DiscoverSubscriptions             bool                          `mapstructure:"discover_subscriptions"`
-	Credentials                       string                        `mapstructure:"credentials"`
-	TenantID                          string                        `mapstructure:"tenant_id"`
```

Comment:
Can you keep just TenantID at this place? Just a nit but this will be clearer when we'll remove the deprecated part

Reply:
Of course